roles/linux_preparation/tasks/recore.yml

```diff
@@ -52,58 +20,41 @@
     - libnss-mdns
     - byobu
     - htop
-    - unzip
     - cpufrequtils
     - pkg-config
     - python3-dev
     - pkg-config
+    - firmware-realtek
     state: present
     autoremove: yes
     autoclean: yes
 
+- name: enable ssh
+  service:
+    name: ssh
+    enabled: yes
+
 - name: enable root login via ssh
   lineinfile:
     path: /etc/ssh/sshd_config
     line: 'PermitRootLogin yes'
     state: present
 
-- name: remove unneeded files
-  file:
-    path: "{{ item }}"
-    state: absent
-  with_items:
-    - /etc/apache2/
-    - /root/.c9
-    - /usr/lib/node_modules
-    - /usr/local/lib/node_modules
-    - /var/lib/cloud9
-    - /var/cache/doc*
-
-- name: create dogtag
-  file:
-    path: '/etc/{{ dogtag }}'
-    state: touch
-
 - name: get date
   command: 'date --iso-8601'
   register: build_date
 
+- name: Register ReFactor version
+  script:
+    cmd: '{{ role_path }}/files/get_refactor_version.sh'
+  register: version
+
 - name: write dogtag
```

Comment:
This will fail if the file doesn't already exist.

Reply:
Which file? The task has just been moved down a bit. It was on top before.

Reply:
The dog tag touch task.